core/platforms/freebsd/bareos-freebsd/bareos.com-common/BareosCommonMakefile

```diff
@@ -75,7 +75,7 @@ CPPFLAGS+=      -I${LOCALBASE}/include
 
 LIB_DEPENDS+=	liblzo2.so:archivers/lzo2 \
 		          libjansson.so:devel/jansson
-LIB_DEPENDS+= libpython2.7.so:lang/python27
+LIB_DEPENDS+= libpython3.7m.so:lang/python37
```

Comment:
how can we build python 2.7 if we don't depend on it?

Reply:
We decided to not support python2.7 on FreeBSD anymore as it will be removed by end of the year.
See:
https://www.freshports.org/lang/python27/: 

DEPRECATED: EOLed upstream
EXPIRATION DATE: 2020-12-31